dashboard/src/src/__tests__/experiments.databasePage.test.js

```diff
@@ -91,18 +102,472 @@ test('Test if experiment trials are loaded', async () => {
+  expect(screen.queryByTitle(/f1f350224ae041550658149b55f6c72a/)).toBeNull();
+  expect(screen.queryByTitle(/f584840e70e38f0cd0cfc4ff1b0e5f2b/)).toBeNull();
+  expect(screen.queryByTitle(/fac3d17812d82ebd17bd771eae2802bb/)).toBeNull();
+  expect(screen.queryByTitle(/fced71d7a9bc1b4fe7c0a4029fe73875/)).toBeNull();
+  expect(screen.queryByTitle(/fd5104909823804b299548acbd089ca6/)).toBeNull();
+  expect(screen.queryByTitle(/ff3adfecab4d01a5e8cb1550cc74b695/)).toBeNull();
+});
+
+test('Test (de)select columns', async () => {
+  const user = userEvent.setup();
+  // Load page
+  const { container } = render(<App />, { wrapper: MemoryRouter });
+  // Switch to database page
+  await user.click(screen.queryByTitle(/Go to experiments database/));
+  // Select an experiment
+  await user.click(
+    await screen.findByText(/2-dim-shape-exp/, {}, global.CONFIG_WAIT_FOR_LONG)
+  );
+  await waitForExperimentToBeLoaded('2-dim-shape-exp');
+  // Check first row for ID, submit time, start time and objective
+  expect(
+    screen.queryByTitle(/0f886905874af10a6db412885341ae0b/)
+  ).toBeInTheDocument();
+  expect(
+    queryByText(container, '2019-11-19 15:41:16.985000')
+  ).toBeInTheDocument();
+  expect(
+    queryByText(container, '2019-11-19 15:41:16.996000')
+  ).toBeInTheDocument();
+  expect(queryByText(container, '-0.7881121864177159')).toBeInTheDocument();
+  // Locate options `submit time`, `objective` and 'select all'
+  const multiSelect = document.getElementById('multiselect-columns');
+  await user.click(multiSelect.querySelector('button.bx--list-box__field'));
+  const optionSubmitTime = queryByText(multiSelect, 'Submit time');
+  const optionObjective = queryByText(multiSelect, 'Objective');
+  const optionSelectAll = queryByText(multiSelect, '(select all)');
+  expect(optionSubmitTime).toBeInTheDocument();
+  expect(optionObjective).toBeInTheDocument();
+  expect(optionSelectAll).toBeInTheDocument();
+  // Deselect column Submit time and check first row
+  await user.click(optionSubmitTime);
+  expect(
+    screen.queryByTitle(/0f886905874af10a6db412885341ae0b/)
+  ).toBeInTheDocument();
+  expect(queryByText(container, '2019-11-19 15:41:16.985000')).toBeNull();
+  expect(
+    queryByText(container, '2019-11-19 15:41:16.996000')
+  ).toBeInTheDocument();
+  expect(queryByText(container, '-0.7881121864177159')).toBeInTheDocument();
+  // Deselect column objective and check first row
+  await user.click(optionObjective);
+  expect(
+    screen.queryByTitle(/0f886905874af10a6db412885341ae0b/)
+  ).toBeInTheDocument();
+  expect(queryByText(container, '2019-11-19 15:41:16.985000')).toBeNull();
+  expect(
+    queryByText(container, '2019-11-19 15:41:16.996000')
+  ).toBeInTheDocument();
+  expect(queryByText(container, '-0.7881121864177159')).toBeNull();
+  // Click to 'select all' and check that all columns are now visible in first row
+  await user.click(optionSelectAll);
```

Comment:
Maybe we should check a few other columns for SelectAll.

Reply:
Done !